### ISSUE TITLE
Unknown files in folder

### ISSUE THREAD

**Issue by paul-seo** [TheCryptoBoss/Crypto-Boss-Feeder#3]
Hi Crypto Boss,

thank you for your ideas of PT Feeder settings. I've just downloaded your last settings and discovered the additional files and folders in 05-superbull folder. 

You can see those on my screenshots. I wonder if it is a mistake or these files and folders are mandatory? 

Thank you.
![screen shot 2018-03-10 at 13 23 13](https://user-images.githubusercontent.com/36458500/37241233-f56f0076-2466-11e8-90d4-cdda5e8d269d.png)

![screen shot 2018-03-10 at 13 29 34](https://user-images.githubusercontent.com/36458500/37241250-36fad70e-2467-11e8-8e06-f01fd6c29605.png)

**Comment by derekpovah** [comment]
@anishpillaimca Yes, the .git folder holds all of the information about the repository, but they generally only belong at root of the project. The extra .git folder here appears to be a second repo nested inside the main one. This is generally bad practice unless it is set up as a submodule. Without seeing the source, there's no way for us to tell if this is the intended behavior.
